ckanext/switzerland/plugin.py

```diff
@@ -159,7 +158,7 @@ def _prepare_package_json(self, pkg_dict):
         # Do not change the resulting dict for API requests and form saves
         # _package_reduce_to_requested_language removes all translation dicts needed
         # to show the form on resource_edit, so we skip it here
-        if sh.request_is_api_request() or toolkit.request.path == "POST":
+        if sh.request_is_api_request() or toolkit.request.method == "POST":
```

Comment:
This was changed by mistake in 2020 - I just noticed it was wrong when adapting this line of code further down.